subprojects/snapshots/src/integTest/groovy/org/gradle/internal/vfs/VirtualFileSystemRetentionIntegrationTest.groovy

```diff
@@ -31,6 +31,7 @@ class VirtualFileSystemRetentionIntegrationTest extends AbstractIntegrationSpec
     def setup() {
         // Make the first build in each test drop the VFS state
         executer.withArgument("-D$VFS_DROP_PROPERTY=true")
+        executer.requireIsolatedDaemons()
```

Comment:
Why is this needed now?

Reply:
It has always been needed: There is no guarantee (without `requireIsolatedDaemons()`) that one test re-uses the same daemon even within a test. As you can see, the flakiness of the test went away after merging my changes: https://e.grdev.net/scans/tests?failures.failureClassification=non_verification&list.offset=0&list.size=50&list.sortColumn=startTime&list.sortOrder=desc&search.buildToolType=gradle&search.buildToolType=maven&search.startTimeMax=1578411071730&search.startTimeMin=1575932400000&search.tags=FunctionalTest&tests.container=org.gradle.internal.vfs.VirtualFileSystemRetentionIntegrationTest&tests.sortField=FAILED&tests.unstableOnly&trends.section=overview&trends.timeResolution=day&viewer.tzOffset=60